src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -158,6 +237,55 @@ class OwlViTObjectDetectionOutput(ModelOutput):
     vision_model_last_hidden_state: Optional[torch.FloatTensor] = None
 
 
+@dataclass
+class OwlViTImageGuidedObjectDetectionOutput(ModelOutput):
+    """
+    Output type of [`OwlViTForObjectDetection.image_guided_detection`].
+
+    Args:
+        loss (`torch.FloatTensor` of shape `(1,)`, *optional*, returned when `labels` are provided)):
```

Comment:
Not sure whether it's relevant to include "loss" and "loss dict" in this output class?

Reply:
True, I'm removing these then.